database/database.go

```diff
@@ -162,6 +195,15 @@ func (d *db) watchMetricsQueue() {
 					delete(d.usage, metricName)
 				}
 			}
+			// In any case, whether the metric is already known or not, we are re-setting the counter.
+			d.metricLastTimeSeen[metricName] = 0
+		}
+		// Finally, we are looping other all counters and remove metrics not seen since a while.
+		for metricName, counter := range d.metricLastTimeSeen {
+			if counter < d.threshold {
```

Comment:
I would have gone with:
```suggestion
			if counter >= d.threshold {
```
in order to have this `continue`

Reply:
I don't get it. If I do like you propose, the `continue` won't be possible as we will have to delete the metric since the threshold has been reached.

Reply:
I did not put everything, what I was saying is:
```golgang
                for metricName, counter := range d.metricLastTimeSeen {
			if counter >= d.threshold {
				d.deleteMetric(metricName)
			}
		}
```